utils/errors/notifier/notifier.go

```diff
@@ -145,6 +145,10 @@ func NotifyWithLevelAndSkip(err error, skip int, level string, rawData ...interf
 		}
 		n.Notified(true)
 	}
+
+	if r, ok := err.(errors.RawExt); ok {
+		rawData = append(r.RawData(), rawData...)
```

Comment:
any specific reason why have put our custom raw data ahead of the other data? basically comparing this vs `rawData = append(rawData, r.RawData())`

Reply:
no specific reason, just append it lol